backend/packages/Upgrade/src/api/services/ExperimentService.ts

```diff
@@ -128,6 +128,12 @@ export class ExperimentService {
     };
   }
 
+  public getMetricQueries(): object {
```

Comment:
Where are you using this? And as this only return value store in `env` you should directly use that.

Reply:
Done. Removed it. Not required.